### DISCUSSION TITLE
Next.js sample fails to start

### DISCUSSION THREAD

**Discussion by thirdnet** [SAP/luigi#3303] [Q&A] [answered]
Hello,

I have been trying to run the Next.js project from the following repository: https://github.com/SAP/luigi/tree/main/core/examples/luigi-example-next

The project works fine when I use `npm run dev`, but I'm encountering an issue when trying to build and start the application using `npm run build` and `npm run start`. 

After running `npm run start`, the browser only displays a white screen and the following error message appears in the console:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'context')
    at eA (286331c2-94f1e2cfff92f22a.js:10:4054)
    at new cx (286331c2-94f1e2cfff92f22a.js:28:389099)
    at 286331c2-94f1e2cfff92f22a.js:28:391600
```
My environment details are as follows:
- Node.js version: v18.16.0
- npm version: 9.5.1

I'm not sure why this is happening and could use some assistance. Has anyone else experienced this issue, or does anyone have suggestions on how to resolve it?

Thank you in advance for your help.

**Comment by JohannesDoberer** [comment]
Hey @thirdnet ,
I'll try it out and come back to you as soon as possible.
https://github.com/SAP/luigi/discussions/3303

**Reply by JohannesDoberer** [reply, marked as answer]
Hello @thirdnet,
I have updated to the latest next.js version, then it works for me.
Here is the corresponding [PR](https://github.com/SAP/luigi/pull/3313/files).
If you try it, please let me know if it works for you too.

Cheers,

Johannes

**Reply by thirdnet** [reply]
Hello @JohannesDoberer 

Thank you so much for your quick response and fix!
I have pulled the 'fix-nextjs-example' branch and I can confirm that the project is now working correctly.
I really appreciate your assistance with this issue. 

Best regards,

thirdnet
